MAINTAINERS

```diff
@@ -0,0 +1,7 @@
+Bryan Boreham, Weaveworks <bryan@weave.works> (@bboreham)
+Chris Marchbanks, FreshTracks <chrism@freshtracks.io> (@csmarchbanks)
+Cody Boggs, FreshTracks <cody@freshtracks.io> (@cboggs)
+Jonathan Lange, Weaveworks <jml@weave.works> (@jml)
+Julius Volz, independent <> (@jvolz)
```

Comment:
Please add my email and the correct GitHub handle:

`Julius Volz, independent <julius.volz@gmail.com> (@juliusv)`